src/plugin.js

```diff
@@ -761,15 +899,350 @@ void main() {
     window.addEventListener('resize', this.handleResize_, true);
     window.addEventListener('vrdisplayactivate', this.handleVrDisplayActivate_, true);
     window.addEventListener('vrdisplaydeactivate', this.handleVrDisplayDeactivate_, true);
+    window.addEventListener('pointerdown', this.handleUserActive_, true);
+
+    // For iOS we need permission for the device orientation data, this will pop up an 'Allow'
```

Comment:
Safari/iOS is only allowing `DeviceMotionEvent.requestPermission` during a user event and not on page load. 

That said, `onDeviceOrientationChange` doesn't seem to do anything with the event data, is it needed?